src/js/models/connectors/Map-Search.js

```diff
@@ -76,9 +76,18 @@ define([
        * model or no Geohash layer in the collection.
        */
       findGeohash: function () {
-        const layers = this.get("layers");
-        if (!layers) return null;
-        let geohashes = layers.getAll("CesiumGeohash");
+        const layerGroups = this.get("layerGroups");
+        if (!layerGroups) return null;
+
+        // TODO: Since only the first Geohash is needed, create a getFirst
+        // function in MapAssets.
+        let geohashes = _.reduce(layerGroups, (memo, layers) => {
+          const geohashes = layers.getAll("CesiumGeohash");
+          if (geohashes && geohashes.length) {
+            memo.push(geohashes);
```

Comment:
@yvonnesjy, nice find! Yes, I think that's a good solution if Map-Search breaks! Rather than removing the toggle component, maybe showing it as greyed out and a user friendly error message with the user hovers over it? Users might be wondering why the functionality disappeared otherwise.

Reply:
kewlkewl I had a quick attempt and it's touching quite a number of files already. since the lack of error handling is not a new issue introduced by this PR, and to not add even more changes to the review, I created a separate issue here: https://github.com/NCEAS/metacatui/issues/2288